hyperspy/misc/eds/utils.py

```diff
@@ -1,6 +1,7 @@
 
 import numpy as np
 import math
+from scipy import constants
```

Comment:
I use avogadro's for the conversions. Probably wasn't needed but now is because the functions have been move to utils.py.